ShurikenMC/Infected-Factions
# connect to db + minor tech debt fixes

**Summary**

in this commit we fix connecting to the db (we have to create a
fat jar with postgres/h2/mysql (supported dbs), also hikaricp
depends on slf4j so we add that into the jar too. I'm not too happy
about this, but it's still only 5MB which isn't too bad considering
all the deps we're throwing into this thing.

**Test Plan**

- `mvn verify` works.
- Spin up a DB, and manually configure config.yml + hikari.properties in the plugin config area of your minecraft server.
- Spin up the minecraft server. It should connect.
